modules/serdes/include/hephaestus/serdes/protobuf/containers.h

```diff
@@ -0,0 +1,66 @@
+//=================================================================================================
+// Copyright (C) 2023-2024 HEPHAESTUS Contributors
+//=================================================================================================
+
+#pragma once
+
+#include <cstddef>
+#include <type_traits>
+#include <vector>
+
+#include <google/protobuf/repeated_field.h>
+#include <google/protobuf/repeated_ptr_field.h>
+
+//=================================================================================================
+// NOTE: In order for namespace resolution to work correctly for container serialization introduced
+// in this file, calls to toProto and fromProto must be prefixed with (heph::)serdes::protobuf::
+//=================================================================================================
+
+namespace heph::serdes::protobuf {
+
+//=================================================================================================
+// Vector (std::vector <-> google::protobuf::RepeatedField)
+//=================================================================================================
+template <typename T, typename ProtoT>
+  requires std::is_arithmetic_v<T>
```

Comment:
either use heph::NumericType or add a proper concept for that.
IMO 
```cpp
template<Arithmetic T, typename ProtoT>
```
is a lot more readable